src/basic/glob-util.c

```diff
@@ -23,15 +56,53 @@ int safe_glob_full(const char *path, int flags, opendir_t opendir_func, char ***
 
         assert(path);
 
+        // TODO: expand braces if GLOB_BRACE is specified but not supported.
+
+#if GLOB_ALTDIRFUNC == 0
+        use_gnu_extension = false;
+#else
+        SET_FLAG(flags, GLOB_ALTDIRFUNC, use_gnu_extension);
```

Comment:
hmm, not supporting GLOB_ALTDIRFUNC is kinda problematic. This stuff is used by tmpfiles, and globbing there shouldn't touch atime of those dirs, but on musl it will if you ignore this here. this means that use of globs will neuter the time-based clean-up stuff...